### ISSUE TITLE
Accessible way of widget rearrangement

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/server#22397]
It should be possible to rearrange widgets by keyboard navigation in addition to drag and drop for accessibility. 

@skjnldsv Is there any magic we can use for this? @jancborchardt mentioned you might know something.

**Comment by skjnldsv** [comment]
On forms we have a drag handle, focusing it with mouse should display two arrows and listen to arrow up/down to reorder :+1: 
We still havent done this but this is actually quite nice as a keyboard navigation.
Some reading https://www.w3.org/wiki/PF/ARIA/BestPractices/DragDrop

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+
